types/src/on_chain_config/aptos_features.rs

```diff
@@ -32,7 +33,7 @@ pub struct Features {
 impl Default for Features {
     fn default() -> Self {
         Features {
-            features: vec![0b00100000],
+            features: vec![0b00100000, 0b00110000],
```

Comment:
Which one does this enable? And why do we need to enable it here?

Reply:
4+8=12, so `GENERIC_ALGEBRAIC_STRUCTURES_BASIC_OPERATIONS`, I presume. But why enable it here?

Reply:
Yes they are `GENERIC_ALGEBRAIC_STRUCTURES_BASIC_OPERATIONS` and `BLS12381_STRUCTURES` respectively.

`GENERIC_ALGEBRAIC_STRUCTURES_BASIC_OPERATIONS` doesn't have to be enabled here but `BLS12381_STRUCTURES` has to, because this is the only way (i know) for rust backend to get feature flags in case of move UTs - enabling in move won't work.

Reply:
Updated PR to take away `GENERIC_ALGEBRAIC_STRUCTURES_BASIC_OPERATIONS` from here.